pkg/services/ngalert/api/tooling/definitions/alertmanager.go

```diff
@@ -804,6 +804,14 @@ func AsGrafanaRoute(r *config.Route) *Route {
 	return gRoute
 }
 
+func (r *Route) ResourceType() string {
+	return "route"
+}
+
+func (r *Route) ResourceID() string {
+	return ""
```

Comment:
There is no unique identifier for a portion of the routing tree. The tree is provisioned in its entirety.

Reply:
These methods make Routes implement `Provisionable`